GOVERNANCE.md

```diff
@@ -0,0 +1,134 @@
+# Serverless Workflow Project Governance
```

Comment:
This doc is from a standard provided by CNCF?

Reply:
No, there is no standard I think as every project might be different depending on community etc. 
I looked a ton of CNCF projects and their governance and thought this would fit ours best at this time.

Reply:
What is required from us for OnBoarding is to have a governance doc. It says that it has to adhere to https://github.com/opengovernance/opengovernance.dev which this one does.